stl/src/awint.h

```diff
@@ -290,45 +290,45 @@ enum wrapKERNEL32Functions {
+using PFNINITONCEEXECUTEONCE             = BOOL(WINAPI*)(PINIT_ONCE, PINIT_ONCE_FN, PVOID, LPVOID*);
+using PFNCREATEEVENTEXW                  = HANDLE(WINAPI*)(LPSECURITY_ATTRIBUTES, LPCWSTR, DWORD, DWORD);
+using PFNCREATESEMAPHOREW                = HANDLE(WINAPI*)(LPSECURITY_ATTRIBUTES, LONG, LONG, LPCWSTR);
+using PFNCREATESEMAPHOREEXW              = HANDLE(WINAPI*)(LPSECURITY_ATTRIBUTES, LONG, LONG, LPCWSTR, DWORD, DWORD);
+using PFNCREATETHREADPOOLTIMER           = PTP_TIMER(WINAPI*)(PTP_TIMER_CALLBACK, PVOID, PTP_CALLBACK_ENVIRON);
+using PFNSETTHREADPOOLTIMER              = VOID(WINAPI*)(PTP_TIMER, PFILETIME, DWORD, DWORD);
+using PFNWAITFORTHREADPOOLTIMERCALLBACKS = VOID(WINAPI*)(PTP_TIMER, BOOL);
+using PFNCLOSETHREADPOOLTIMER            = VOID(WINAPI*)(PTP_TIMER);
+using PFNCREATETHREADPOOLWAIT            = PTP_WAIT(WINAPI*)(PTP_WAIT_CALLBACK, PVOID, PTP_CALLBACK_ENVIRON);
+using PFNSETTHREADPOOLWAIT               = VOID(WINAPI*)(PTP_WAIT, HANDLE, PFILETIME);
+using PFNCLOSETHREADPOOLWAIT             = VOID(WINAPI*)(PTP_WAIT);
+using PFNFLUSHPROCESSWRITEBUFFERS        = VOID(WINAPI*)();
+using PFNFREELIBRARYWHENCALLBACKRETURNS  = VOID(WINAPI*)(PTP_CALLBACK_INSTANCE, HMODULE);
+using PFNGETCURRENTPROCESSORNUMBER       = DWORD(WINAPI*)();
+using PFNCREATESYMBOLICLINKW             = BOOLEAN(WINAPI*)(LPCWSTR, LPCWSTR, DWORD);
+using PFNGETTICKCOUNT64                  = ULONGLONG(WINAPI*)();
+using PFNGETFILEINFORMATIONBYHANDLEEX    = BOOL(WINAPI*)(HANDLE, FILE_INFO_BY_HANDLE_CLASS, LPVOID, DWORD);
+using PFNSETFILEINFORMATIONBYHANDLE      = BOOL(WINAPI*)(HANDLE, FILE_INFO_BY_HANDLE_CLASS, LPVOID, DWORD);
+using PFNGETSYSTEMTIMEPRECISEASFILETIME  = VOID(WINAPI*)(LPFILETIME);
+using PFNINITIALIZECONDITIONVARIABLE     = VOID(WINAPI*)(PCONDITION_VARIABLE);
+using PFNWAKECONDITIONVARIABLE           = VOID(WINAPI*)(PCONDITION_VARIABLE);
+using PFNWAKEALLCONDITIONVARIABLE        = VOID(WINAPI*)(PCONDITION_VARIABLE);
+using PFNSLEEPCONDITIONVARIABLECS        = BOOL(WINAPI*)(PCONDITION_VARIABLE, PCRITICAL_SECTION, DWORD);
+using PFNINITIALIZESRWLOCK               = VOID(WINAPI*)(PSRWLOCK);
+using PFNACQUIRESRWLOCKEXCLUSIVE         = VOID(WINAPI*)(PSRWLOCK);
+using PFNTRYACQUIRESRWLOCKEXCLUSIVE      = BOOLEAN(WINAPI*)(PSRWLOCK);
+using PFNRELEASESRWLOCKEXCLUSIVE         = VOID(WINAPI*)(PSRWLOCK);
+using PFNSLEEPCONDITIONVARIABLESRW       = BOOL(WINAPI*)(PCONDITION_VARIABLE, PSRWLOCK, DWORD, ULONG);
+using PFNCREATETHREADPOOLWORK            = PTP_WORK(WINAPI*)(PTP_WORK_CALLBACK, PVOID, PTP_CALLBACK_ENVIRON);
+using PFNSUBMITTHREADPOOLWORK            = VOID(WINAPI*)(PTP_WORK);
+using PFNCLOSETHREADPOOLWORK             = VOID(WINAPI*)(PTP_WORK);
+using PFNCOMPARESTRINGEX = int(WINAPI*)(LPCWSTR, DWORD, LPCWSTR, int, LPCWSTR, int, LPNLSVERSIONINFO, LPVOID, LPARAM);
+using PFNGETLOCALEINFOEX = int(WINAPI*)(LPCWSTR, LCTYPE, LPWSTR, int);
+using PFNLCMAPSTRINGEX   = int(WINAPI*)(LPCWSTR, DWORD, LPCWSTR, int, LPWSTR, int, LPNLSVERSIONINFO, LPVOID, LPARAM);

```

Comment:
Are we not lining up the `=` because it causes the line to exceed 120 characters?

Reply:
Yes, that's almost certainly what clang-format is doing here.

Reply:
When clang-format works, it really generates nice results. Now if only it understood variable templates and concepts...

Reply:
> When clang-format works, it really generates nice results.

Isn't this [tautological](https://xkcd.com/703/)?

Reply:
Related: Should we have autolink support for XKCD-XXXX?